pkg/kubelet/kubelet.go

```diff
@@ -40,9 +40,8 @@ import (
 	semconv "go.opentelemetry.io/otel/semconv/v1.12.0"
 	"go.opentelemetry.io/otel/trace"
 	"k8s.io/client-go/informers"
-	"k8s.io/mount-utils"
-
 	utilfs "k8s.io/kubernetes/pkg/util/filesystem"
+	"k8s.io/mount-utils"
 	netutils "k8s.io/utils/net"
```

Comment:
why changing it to other imports group. is it automatic sorting of imports?

Reply:
@mxpv, your formatter or LSP of your editor might be using different order than Kubernetes.

Reply:
Ya, IDEA is making my life harder here :) Give me a moment to fix it.